code/__DEFINES/achievements.dm

```diff
@@ -4,102 +4,102 @@
+#define MAFIA_MEDAL_FUGITIVE "Fugitive"
+#define MAFIA_MEDAL_OBSESSED "Obsessed"
+#define MAFIA_MEDAL_CLOWN "Clown"
 
 //Mafia medal hub IDs (misc stuff)
-#define MAFIA_MEDAL_HATED		"Universally Hated"
-#define MAFIA_MEDAL_CHARISMATIC	"Charismatic"
-#define MAFIA_MEDAL_VIP			"VIP"
+#define MAFIA_MEDAL_HATED "Universally Hated"
+#define MAFIA_MEDAL_CHARISMATIC "Charismatic"
+#define MAFIA_MEDAL_VIP "VIP"
 
 //Boss medals
 
 // Medal hub IDs for boss medals (Pre-fixes)
-#define BOSS_MEDAL_ANY		  "Boss Killer"
-#define BOSS_MEDAL_MINER	  "Blood-drunk Miner Killer"
+#define BOSS_MEDAL_ANY   "Boss Killer"
+#define BOSS_MEDAL_MINER   "Blood-drunk Miner Killer"
 #define BOSS_MEDAL_FROSTMINER "Demonic-frost Miner Killer"
 #define BOSS_MEDAL_BUBBLEGUM  "Bubblegum Killer"
-#define BOSS_MEDAL_COLOSSUS	  "Colossus Killer"
-#define BOSS_MEDAL_DRAKE	  "Drake Killer"
+#define BOSS_MEDAL_COLOSSUS   "Colossus Killer"
+#define BOSS_MEDAL_DRAKE   "Drake Killer"
 #define BOSS_MEDAL_HIEROPHANT "Hierophant Killer"
-#define BOSS_MEDAL_LEGION	  "Legion Killer"
-#define BOSS_MEDAL_TENDRIL	  "Tendril Exterminator"
+#define BOSS_MEDAL_LEGION   "Legion Killer"
+#define BOSS_MEDAL_TENDRIL   "Tendril Exterminator"
 #define BOSS_MEDAL_SWARMERS   "Swarmer Beacon Killer"
 #define BOSS_MEDAL_WENDIGO    "Wendigo Killer"
 #define BOSS_MEDAL_KINGGOAT   "King Goat Killer"
 
-#define BOSS_MEDAL_MINER_CRUSHER	  	"Blood-drunk Miner Crusher"
-#define BOSS_MEDAL_FROSTMINER_CRUSHER	"Demonic-frost Miner Crusher"
-#define BOSS_MEDAL_BUBBLEGUM_CRUSHER  	"Bubblegum Crusher"
-#define BOSS_MEDAL_COLOSSUS_CRUSHER	  	"Colossus Crusher"
-#define BOSS_MEDAL_DRAKE_CRUSHER	  	"Drake Crusher"
-#define BOSS_MEDAL_HIEROPHANT_CRUSHER 	"Hierophant Crusher"
-#define BOSS_MEDAL_LEGION_CRUSHER	 	"Legion Crusher"
-#define BOSS_MEDAL_SWARMERS_CRUSHER		"Swarmer Beacon Crusher"
-#define BOSS_MEDAL_WENDIGO_CRUSHER		"Wendigo Crusher"
-#define BOSS_MEDAL_KINGGOAT_CRUSHER		"King Goat Crusher"
+#define BOSS_MEDAL_MINER_CRUSHER "Blood-drunk Miner Crusher"
+#define BOSS_MEDAL_FROSTMINER_CRUSHER "Demonic-frost Miner Crusher"
+#define BOSS_MEDAL_BUBBLEGUM_CRUSHER "Bubblegum Crusher"
+#define BOSS_MEDAL_COLOSSUS_CRUSHER "Colossus Crusher"
+#define BOSS_MEDAL_DRAKE_CRUSHER "Drake Crusher"
+#define BOSS_MEDAL_HIEROPHANT_CRUSHER "Hierophant Crusher"
+#define BOSS_MEDAL_LEGION_CRUSHER "Legion Crusher"
+#define BOSS_MEDAL_SWARMERS_CRUSHER "Swarmer Beacon Crusher"
+#define BOSS_MEDAL_WENDIGO_CRUSHER "Wendigo Crusher"
+#define BOSS_MEDAL_KINGGOAT_CRUSHER "King Goat Crusher"
 
 // Medal hub IDs for boss-kill scores
-#define BOSS_SCORE 	         "Bosses Killed"
-#define MINER_SCORE 		 "BDMs Killed"
-#define FROST_MINER_SCORE 	 "DFMs Killed"
-#define BUBBLEGUM_SCORE 	 "Bubblegum Killed"
-#define COLOSSUS_SCORE 	     "Colossus Killed"
-#define DRAKE_SCORE 	     "Drakes Killed"
-#define HIEROPHANT_SCORE 	 "Hierophants Killed"
-#define LEGION_SCORE 	     "Legion Killed"
+#define BOSS_SCORE          "Bosses Killed"
```

Comment:
Guess it didn't catch all cases, chiefly that of multiple spaces.
Can you address that as well on the contribution guidelines change?

Reply:
Do you not feel the wording of the guidelines is adequate enough? I wrote "tabs/spaces".

Reply:
Nevermind, the wording is fine.
Maybe we should lint for this as well.